fendermint/app/src/settings.rs

```diff
@@ -36,6 +36,16 @@ pub struct DbSettings {
     pub state_hist_size: u64,
 }
 
+#[derive(Debug, Deserialize)]
+pub struct FvmSettings {
+    /// Overestimation rate applied to gas estimations to ensure that the
+    /// message goes through
+    pub gas_overestimation_rate: f64,
+    /// Gas search step increase used to find the optimal gas limit.
+    /// It determines how fine-grained we want the gas estimation to be.
+    pub gas_search_step: f64,
+}
```

Comment:
You'll need to add default values to https://github.com/consensus-shipyard/fendermint/blob/b1493d05b46891c72f43653d747b2ace790c0958/fendermint/app/config/default.toml

Reply:
Hmm... I did. I don't know why they haven't been pushed. Let me check